src/Sulu/Bundle/AdminBundle/Resources/js/views/List/List.js

```diff
@@ -41,16 +41,24 @@ class List extends React.PureComponent<ViewProps> {
 
     handleEditClick = (rowId) => {
         const {router} = this.props;
-        router.navigate(router.route.options.editLink, {uuid: rowId});
+        router.navigate(router.route.options.editRoute, {id: rowId});
     };
 
     render() {
+        const {
+            route: {
+                options: {
+                    title,
+                    editRoute,
+                },
+            },
+        } = this.props.router;
         return (
             <div>
-                <h1>List</h1>
+                {title && <h1>{translate(title)}</h1>}
```

Comment:
This made me ask myself where translations should be allowed... I think translating at this location is fine, but I am not so sure anymore about the `Pagination` component. It's actually a pretty simple component, which accesses a service. Is this a good idea? Shouldn't we say translations are only done in the views, since it is accessing a service? Or do we say translations are an exception of the rule?

Reply:
remove services from components.